src/v2/Apps/Conversation/Components/ConversationHeader.tsx

```diff
@@ -1,61 +1,57 @@
-import { FC } from "react";
+import { FC } from "react"
 import {
   ArrowLeftIcon,
   Box,
   Flex,
-  FlexProps,
   Text,
   Separator,
+  breakpoints,
 } from "@artsy/palette"
 import styled from "styled-components"
 import { themeGet } from "@styled-system/theme-get"
 
 import { RouterLink } from "v2/System/Router/RouterLink"
-import { Media } from "v2/Utils/Responsive"
 import { DetailIcon, DetailsProps } from "./DetailsHeader"
 
 export const LARGE_SCREEN_HEADER_HEIGHT = "85px"
-const SMALL_SCREEN_HEADER_HEIGHT = "55px"
-
-interface BorderedFlexProps extends FlexProps {
-  bordered?: boolean
-}
-const BorderedFlex = styled(Flex)<BorderedFlexProps>`
-  ${props =>
-    props.bordered
-      ? `border-right: 1px solid ${themeGet("colors.black10")};`
-      : ""}
-  height: 100%;
-`
+export const SMALL_SCREEN_HEADER_HEIGHT = "55px"
 
 const SmallConversationHeaderContainer = styled(Flex)`
-  position: fixed;
-  top: 59px;
+  position: relative;
+  top: 0;
   left: 0;
   right: 0;
   border-bottom: 1px solid ${themeGet("colors.black10")};
   background: white;
   z-index: 1;
+
+  @media (min-width: ${breakpoints.md}) {
+    display: none;
+  }
+`
+
+const LargeConversationHeaderContainer = styled(Flex)`
+  display: none;
+
+  @media (min-width: ${breakpoints.md}) {
+    display: initial;
+  }
 `
 
 export const ConversationHeader: FC<ConversationHeaderProps> = props => {
   const { partnerName, showDetails, setShowDetails } = props
   return (
     <>
-      <Media between={["xs", "sm"]}>
-        <SmallConversationHeader
-          showDetails={showDetails}
-          setShowDetails={setShowDetails}
-          partnerName={partnerName}
-        />
-      </Media>
-      <Media greaterThan="sm">
-        <LargeConversationHeader
-          showDetails={showDetails}
-          setShowDetails={setShowDetails}
-          partnerName={partnerName}
-        />
-      </Media>
+      <SmallConversationHeader
```

Comment:
I guess for this case the ideal usage would be
```jsx
<Media lessThan="md">
  <SmallConversationHeader />
</Media>
<Media greaterThanOrEqual="md">
  <LargeConversationHeader /> 
</Media>
```
then you don't need that breakpoints control inside the styled-components and I think that won't cause any visual issues like the one we had in the routes file.
Based on https://artsy.github.io/blog/2019/05/24/server-rendering-responsively/

Reply:
I _think_ that this was required because of some quirk in how our `Media` component works. I'll double check before merging!

Reply:
Ok, so I checked this today and thought it seemed fine with the Media components but now I think i remember the source of the bug- these are flex items (their parent is a `Flex`) and the styles in the actual `_ConversationHeader` components styles are not being applied as expected. I'll triple check this tomorrow before merging!

Reply:
Thanks for the effort here, wasn't a major thing but good to keep consistency with responsiveness usage around!